http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpDockerDAO.scala

```diff
@@ -210,9 +236,9 @@ final case class ParsedImage(registry: ContainerRegistry,
 }
 
 // API response models
-final case class Token(token: String)
+final case class Token(token: String) extends AnyVal
 final case class ManifestConfig(mediaType: String, size: Int, digest: String)
-final case class ContainerConfig(image: String, env: List[String])
```

Comment:
We haven't been using `image` so removed this `case class` and stopped trying to parse that field from the JSON, also because the particular user image doesn't contain that field at all.